packages/victory-axis/src/helper-methods.js

```diff
@@ -257,21 +340,28 @@ const getGridEdge = (props, calculatedValues) => {
   return { x, y };
 };
 
-const getGridOffset = (props, calculatedValues, offset) => {
-  const { padding, orientation } = calculatedValues;
+const getGridOffset = (calculatedValues, offset) => {
+  const { padding, orientation, crossAxis } = calculatedValues;
   const xPadding = orientation === "right" ? padding.right : padding.left;
   const yPadding = orientation === "top" ? padding.top : padding.bottom;
   return {
-    x: props.crossAxis ? offset.x - xPadding : 0,
-    y: props.crossAxis ? offset.y - yPadding : 0
+    x: crossAxis ? offset.x - xPadding : 0,
+    y: crossAxis ? offset.y - yPadding : 0
   };
 };
 
 const getLayoutProps = (modifiedProps, calculatedValues) => {
-  const offset = getOffset(modifiedProps, calculatedValues);
+  let offset;
+  if (modifiedProps.standalone) {
```

Comment:
Are you using `standalone` as a check for whether the axis is a child of `VictoryChart`? That's not a perfect check for that, unfortunately. Maybe it's worth adding a flag for that, but for now, a better check would be the existence of x and y domain values.

Reply:
I was, yes.  Wasn't 100% sure of that, good to know. I'll switch it to domain.

Reply:
done